data/views/node.2.0.json

```diff
@@ -1,4 +1,13 @@
 {
+<%
+    var _ = injector.get('_');
```

Comment:
@srinia6 I know that this is done in other places as well, so it is probably outside the scope of this PR, but we should change the template render context to just include the injectables we actually need (lodash in this case) rather than the whole injector.  This will prevent a malicious user from posting a view and getting full access to all modules.